README.md

```diff
@@ -389,6 +389,7 @@ Basic steps:
 
 * `mvn clean install` to run all of the project's tests and install the library in your local repo
 * `mvn -Dquickly` to install the library skipping all tests
+* `mvn -Pdev <...goals>` to disable linters and enforcers during development
```

Comment:
I haven't tried this yet, but since you added an activation property, I think you can write, for consistency with `-Dquickly`: 

```suggestion
* `mvn -Ddev <...goals>` to disable linters and enforcers during development
```

Reply:
it's the same, no particular preference from my side.